designs/administrator/future-openshift/saved-searches/index.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Saved filters
+
+The Search page will be updated to allow users to save searches.
+
+## General flow
+
+![1](img/saved1.png)
+* If no filters have been saved, the **Saved filters** dropdown will have a small empty state.
```

Comment:
This is just an opinion that relates to the Search page in general, but I sort of feel like this pages empty state should  be actionable to get something there in a hurry, especially since the "Resources" dropdown is ever so slightly more mixed in now. I think the MCM team showed that they list out the saved filters as tiles in the empty state so you can just click one and see the matching resources? Maybe if no filters are saved have common suggested filters like "Show all pods" or something? This is just a random thought but figured I'd float out there.

Reply:
Small nit. but maybe clearer if empty state says something like "Create and save filters to access them here".

Reply:
I like that copy @dtaylor113, makes sense to me. And yeah that's a good point @itsptk . I'm curious if PF has any empty states that do anything like that – it would be kind of cool to show a set of default filters users can easily click.